lib/internal/Magento/Framework/Serialize/Serializer/JsonHexTag.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
It would be best to cover newly introduced serializer with a test, as it seems to be a critical part of the framework

Reply:
Added a test